lib/eldap/src/eldap.erl

```diff
@@ -309,6 +356,11 @@ simple_bind(Handle, Dn, Password) -> return_value()
 Authenticate the connection using simple authentication.
 """.
 -doc(#{since => <<"OTP R15B01">>}).
+-spec simple_bind(Handle, Dn, Password) -> ok | {error, Reason} when
```

Comment:
According to the old doc this function can also return "`{ok, {referral, referrals()}}`".